x-pack/solutions/observability/plugins/synthetics/server/alert_rules/status_rule/status_rule_executor.ts

```diff
@@ -567,6 +567,16 @@ export class StatusRuleExecutor {
       });
     }
 
+    const grouping: Record<string, unknown> = {
+      monitor: { id: monitorSummary.monitorId, config_id: monitorSummary.configId },
+    };
+    if (locationIds.length === 1) {
+      grouping.location = { id: locationIds[0] };
```

Comment:
if there are multiple locations, can we store an array of locations in `grouping.location.id`? I think we should be able to store array of string and string in same field, but we will need to test it to make sure it works. Also, adding tests for multiple locations and single location would be helpful.